recipe/meta.yaml

```diff
@@ -61,21 +59,20 @@ requirements:
         - {{ cdt('libxau') }}                # [linux]
         - {{ cdt('libxxf86vm') }}            # [linux]
     host:
-        - libblas
-        - libcblas
-        - liblapack
-        - libglu                  # [linux]
-        - freeglut                # [linux]
-        - xorg-libxmu             # [linux]
+        - openblas  # [win]
+        - libblas  # [not win]
+        - liblapack  # [not win]
+        - libglu  # [linux]
+        - freeglut  # [linux]
```

Comment:
This is available cross platform via conda forge now, but maybe we can just change that in a future PR.

Reply:
I agree; this could take some time to resolve, given that Macs provide Glut. We would want to test the actual visualizer behavior on Mac.